EventFilter/EcalRawToDigi/plugins/EcalRawToDigiGPU.cc

```diff
@@ -33,6 +33,7 @@ class EcalRawToDigiGPU : public edm::stream::EDProducer<edm::ExternalWork> {
 
   cms::cuda::ContextState cudaState_;
 
+  const uint32_t nbytes_per_fed_max_;
```

Comment:
could you change `nbytes_per_fed_max_` to `maxFedSize_` for consistency with the other variables ?